Bulldozer.CSV/Maps/Financial.cs

```diff
@@ -1448,14 +1448,11 @@ private int LoadScheduledTransaction( CSVInstance csvData )
                     }
                     else
                     {
-                        throw new ArgumentOutOfRangeException( $"Cannot find person alias with person key '{rowPersonKey}' for scheduled transaction key {rowTransactionKey}" );
+                        LogException( "InvalidPersonAlias", string.Format( "Cannot find person alias with person key '{0}' for scheduled transaction key {1}", rowPersonKey, rowTransactionKey ) );
+                        currentTransaction = new FinancialScheduledTransaction();
+                        continue;
                     }
 
-                    currentTransaction.CreatedDateTime = ParseDateOrDefault( row[ScheduledTransactionCreatedDate], ImportDateTime );
```

Comment:
Should these have been removed, or should they be inside the personalias check logic?